plugins/lsp/schemastore.nix

```
@@ -0,0 +1,151 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.vim-plugin.mkVimPlugin config {
+  name = "schemastore";
+  originalName = "SchemaStore.nvim";
+  defaultPackage = pkgs.vimPlugins.SchemaStore-nvim;
+
+  maintainers = [ maintainers.GaetanLepage ];
+
+  extraOptions =
+    let
+      schemaOpts =
+        let
+          schemaEntry = types.submodule {
+            options = {
+              name = mkOption {
+                type = types.str;
+                description = "The name of the schema.";
+              };
```

This needs `maybeRaw`, so either:

```suggestion
              name = helpers.mkNullOrStr "The name of the schema.";
```

Or (if you don't want `nullOr`):

```suggestion
              name = mkOption {
                type = with helpers.nixvimTypes; maybeRaw str;
                description = "The name of the schema.";
              };
```